win32/src/file_win32.c

```diff
@@ -60,4 +41,472 @@ static void on_file_io_complete_win32(
     ULONG io_result,
     ULONG_PTR number_of_bytes_transferred,
     PTP_IO io
-);
\ No newline at end of file
+);
+
+static VOID NTAPI on_close_threadpool_group_member(
+     PVOID object_context, /*what was passed @  CreateThreadpoolIo*/
+     PVOID cleanup_context /*what was passed @  CloseThreadpoolCleanupGroupMembers*/
+)
+{
+    (void)object_context;
+    (void)cleanup_context;
+}
+
+IMPLEMENT_MOCKABLE_FUNCTION(
+    ,
+    FILE_HANDLE, file_create,
+    EXECUTION_ENGINE_HANDLE, execution_engine,
+    const char*, full_file_name,
+    FILE_REPORT_FAULT, user_report_fault_callback,
+    void*, user_report_fault_context
+)
+{
+    FILE_HANDLE handle = (FILE_HANDLE)NULL;

```

Comment:
>FILE_HANDLE [](start = 26, length = 11)

Cast not needed.

Reply:
done

---
In reply to: [456705159](https://github.com/Azure/azure-c-pal/pull/10#discussion_r456705159) [](ancestors = 456705159)